### ISSUE TITLE
Code cell not updated from recent fix

### ISSUE THREAD

**Issue by joeldmott** [learn-co-curriculum/dsc-t-tests#6]
## Link to Canvas

https://learning.flatironschool.com/courses/6535/assignments/239175?module_item_id=562067

## Issue Subtype

- [x] Master branch code
- [ ] Solution branch code
- [ ] Code tests
- [x] Layout/rendering issue
- [ ] Instructions unclear
- [ ] Other (explain below)

### Source

```
sns.set(color_codes=True)
sns.histplot(sample, kde=True, bins=5);
```

### Concern

This is in code cell 9. It looks like a recent fix added 'color='darkblue' to the kde plot, but that isn't in the current version of the notebook, so it runs the previous version, which throws an error.

## What OS Are You Using?

- [ ] OS X
- [x] Windows
- [ ] WSL
- [ ] Linux
- [ ] Saturn Cloud from Canvas

**Comment by jessepisel** [comment]
Thanks for opening this issue with us @joeldmott :wave:. Can you update your version of matplotlib and try running the notebook again? 

To update run the following command in a terminal:
```
conda activate learn-env && conda install conda-forge::matplotlib[version='>=3.7']
```

This will ensure your version of matplotlib in the `learn-env` environment is at least version 3.7 of matplotlib and will work with the new keyword argument.

**Comment by joeldmott** [comment]
Hi Jesse,

Sorry, I tried that and I got this output:
[image: image.png]
So, it looks like I have to try initializing my shell. May I ask if you
know what I'd do next if I don't know my shell name? I tried like "conda
init ***@***.***" and other stuff just based on what I see in my
terminal window, but I don't know what's meant by "shell name." Sorry, I'm
unfamiliar with the terminal outside of changing directories and doing
basic git commands.

- Joel

**Comment by jessepisel** [comment]
Hi Joel, can you add the image to GitHub so I can see what the output looks like? It did not come through via email. Happy to give this a shot and get it functioning correctly for you!

**Comment by joeldmott** [comment]
Sure thing, so I tried running that command again to see if maybe it'd work. At first, I was hopeful because, instead of that error message in the terminal, it looked like it was going through a bunch of files and maybe updating them. Afterward, I restarted my computer to see if ti worked, but unfortunately it still threw the same error in the Jupyter notebook. Here is that error:
![error message](https://github.com/learn-co-curriculum/dsc-t-tests/assets/51928528/235675f6-1712-4766-bd24-31216a97e838)
And here is what the terminal looks like when I try re-running the update command. This is just snapshot of the long process it runs through, but the errors persist after it finishes whatever it's doing here:
![image](https://github.com/learn-co-curriculum/dsc-t-tests/assets/51928528/3d7a2cf9-6c6e-4df3-a4b4-5d92f115d43c)

**Comment by jessepisel** [comment]
That's kind of strange that it is not solving the environment with conda. The last option is to try and upgrade matplotlib with pip to deal with incompatibility issues. After you activate your `learn-env` can you run `pip install matplotlib --upgrade` to upgrade matplotlib in your `learn-env` and give that a shot?

**Comment by joeldmott** [comment]
Hi Jesse, sorry for the late reply. I tried the `pip install matplotlib --upgrade` solution while in the learning environment, but I still got the same error. This same error happens again in Phase 3, Week 1: Inferential Modeling - Multiple Linear Regression & Transformation of Linear Models, code cell #6. This is almost certainly because it also involves a Seaborn histplot. 

I think it has something to do with the color param because when I add a specific color to the first of two histplot code lines (shown below), it worked. When I deleted both color params from these lines, it still threw the error.

![image](https://github.com/learn-co-curriculum/dsc-t-tests/assets/51928528/dcd639b5-0e95-4a1f-a2e0-46a83b46297e)

When I look up the[ Seaborn histplot documentation for the color param](https://seaborn.pydata.org/generated/seaborn.histplot.html), it says "Single color specification for when hue mapping is not used. Otherwise, the plot will try to hook into the matplotlib property cycle."
